ARMeilleure/Translation/PTC/Ptc.cs

```diff
@@ -833,15 +840,22 @@ void TranslateFuncs()
 
         private static void TranslationLogger(object state)
         {
-            const int refreshRate = 1; // Seconds.
+            const int refreshRate = 100; // ms
 
             int profiledFuncsToTranslateCount = (int)state;
 
-            do
+            try
+            {
+                do
+                {
+                    PtcTranslationProgressChanged(_translateCount, profiledFuncsToTranslateCount);
```

Comment:
```suggestion
                    PtcTranslationProgressChanged?.Invoke(_translateCount, profiledFuncsToTranslateCount);
```
From what I recall, the event will be null if theres no listeners, so it shouldn't attempt to do the call in this case.

Reply:
Thanks for catching! `myEvent += myHandler` works without needing a null check and this inconsistency bothers me.